app/components/wallet/backup-recovery/WalletRecoveryPhraseEntryDialog.js

```diff
@@ -95,10 +95,9 @@ export default class WalletRecoveryPhraseEntryDialog extends Component<Props> {
     const actions = [];
 
     actions.push({
-      className: isSubmitting ? styles.isSubmitting : null,
-      label: intl.formatMessage(messages.buttonLabelConfirm),
-      onClick: onFinishBackup,
-      disabled: !canFinishBackup,
+      label: intl.formatMessage(messages.buttonLabelRemoveLast),
```

Comment:
Probably this is what is causing the test to fail. I think you have to keep `onFinishBackup` somehow